<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Solution</class>
 <widget class="QWidget" name="Solution">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="TableWidget" name="tableView">
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>200</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>600</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="controlBox">
     <property name="minimumSize">
      <size>
       <width>60</width>
       <height>168</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>168</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QPushButton" name="previousStep">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Шаг назад&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/new/icons/resources/previous.png</normaloff>:/new/icons/resources/previous.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="nextStep">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Шаг вперед&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/new/icons/resources/next.png</normaloff>:/new/icons/resources/next.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="autoSolution">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="toolTip">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Автоматическое решение&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/new/icons/resources/auto.png</normaloff>:/new/icons/resources/auto.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableWidget</class>
   <extends>QTableView</extends>
   <header>tablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
